utils/giftRequests.ts

```diff
@@ -24,8 +24,8 @@ export async function getGift(id: string) {
  *
  * @param newObject a new object of a gift with type Gift that will added to the server
  */
-export async function createGift(newObject: Gift) {
-  await axios.post(`${baseURL}/`, newObject);
+export async function createGift(newObject: CreateGift) {
+  return await axios.post(`${baseURL}`, newObject);
```

Comment:
Palautetaan Gift-objekti tässä ja `updateGift`issä. Delete ei yleensä palauta mitään.
```suggestion
  return (await axios.post(`${baseURL}`, newObject)).data as Gift;
```